docs/changes/coordinates/12501.feature.rst

```diff
@@ -0,0 +1,2 @@
+Supported versions of JPL ephemeris files can now be accessed from the
```

Comment:
This maybe is slightly misleading, as one could always give the full URL to the ephemeris, so it should really be "supported short-cuts to..."